src/wovnio/html/HtmlConverter.php

```diff
@@ -38,18 +39,88 @@ public function __construct($html, $encoding, $token, $store, $headers)
     $this->headers = $headers;
   }
 
+  public function insertSnippetAndHreflangTags() {
+    $this->html = $this->insertSnippet($this->html);
+    $this->html = $this->insertHreflangTags($this->html);
+    $marker = new HtmlReplaceMarker();
+    return array($this->html, $marker);
+  }
+
   /**
    * Convert to appropriate HTML to send to Translation API
    * e.g.) remove wovn-ignore content
    *
    * @return array converted html and HtmlReplaceMarker
    */
-  public function insertSnippetAndHreflangTags()
+  public function convertToAppropriateForApiBody()
```

Comment:
`convertToAppropriate*Format*ForApiBody` might be better imo

Reply:
thanks, done